Add project-directory option to the Init task (#29992)
gradle/gradle: platforms/documentation/docs/src/docs/userguide/reference/core-plugins/build_init_plugin.adoc

gradle init --type java-library --dsl kotlin
```

> You can change the directory where the project will be generated using the `--into` option. It defaults to the directory where the `init` task is run.

Reviewer: ```suggestion
You can change the directory for the generated project using the `--into` option. It defaults to the directory where the `init` task is run.
```
